### ISSUE TITLE
Investigate Integration with tensorflow 

### ISSUE THREAD

**Issue by parmidaatg** [ElementAI/baal#137]
**Is your feature request related to a problem? Please describe.**
As per conversation with @tboquet,  this feature could help him working with BaaL in his research at Unity. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Dref360** [comment]
What's missing?

**Comment by tboquet** [comment]
We might want to have a look at how the transformers repo at hugging works w.r.t having 2 or more libraries work in the future:
- https://github.com/huggingface/transformers/blob/1a3deae8202bb9d64ef9419ca18f7392255b9005/src/transformers/file_utils.py#L80-L111
- https://github.com/huggingface/transformers/blob/1a3deae8202bb9d64ef9419ca18f7392255b9005/src/transformers/file_utils.py#L80-L111
- https://github.com/huggingface/transformers/blob/75e63dbf70d566f6be122b16bb2fb22e1bb6e558/src/transformers/models/mt5/__init__.py#L48-L72

But we can do without right?
There would be a model wrapper to implement and the dirichlet calibration?

**Comment by parmidaatg** [comment]
yes mainly model_wrapper, and patch_module (this is the main one which changes droput layer) and dirichlet. the heuristics are numpy so we are good
